packages/mdc-switch/_mixins.scss

```diff
@@ -192,6 +192,29 @@
   }
 }
 
+// Opacity customizations apply to both on and off states to ensure symmetry.
+// Both checked and unchecked selectors are included to preserve specificity.
+@mixin mdc-switch-ripple-hover-opacity($opacity, $query: mdc-feature-all()) {
+  &:not(.mdc-switch--checked) .mdc-switch__thumb-underlay,
```

Comment:
Since this selectors covers both checked and unchecked state, we can just use `.mdc-switch__thumb-underlay` as a selector?

Reply:
This addresses specificity concerns - the base mixins for switch customize toggled-on and toggled-off colors/opacities for ripple separately, so to override them the same level of specificity is maintained

Reply:
Can we just use `&.mdc-switch .mdc-switch__thumb-underlay` in that case replace two selectors? Also adding a code comment would be helpful.

Reply:
Yes, that's a better solution! Also added comment.